model/Basket/Concessions/AddOns.py

```diff
@@ -0,0 +1,59 @@
+import six
+from abc import ABCMeta
+from Concessions.Popcorn import Popcorn
+
+@six.add_metaclass(ABCMeta)
+class AddOns(Popcorn):
```

Comment:
does this class need to extend popcorn? should it not be the other way around?

Reply:
Like addPopcorn(AddOns)

Reply:
No you get popcorn and then addOns is the decorator class for adding more concessions

Reply:
ohh so you can only get add ons if you order popcorn?

Reply:
@eoinc134

Reply:
ya you start with popcorn and add whatever else you want